src/components/Header/Header.jsx

```diff
@@ -1,18 +1,36 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+
 import { Button } from '../../common/Button';
 import { Logo } from './components/Logo';
 import { BUTTON_LOGOUT } from '../../constants';
 
 import './header.css';
 
-function Header() {
+function Header({ userName }) {
+	const location = useLocation();
+	const navigate = useNavigate();
+
+	const logout = () => {
+		localStorage.clear();
+		navigate('/login');
+	};
```

Comment:
I think it will be better to only remove the current user and not clear while local storage

Reply:
I knew you would make a remark here